openassessment/xblock/static/sass/themes/edx/_utilities.scss

```diff
@@ -10,47 +10,11 @@
 // --------------------
 $edx-blue: rgb(58, 162, 224);
 $edx-blue-l1: tint($edx-blue,20%);
-$edx-blue-l2: tint($edx-blue,40%);
```

Comment:
Deleting a lot of variables that aren't used (or have other values in _variables.scss).

When the code was originally written, there was an attempt to have xblock-general colors and then edX overrides. No other repo is using this though, and at this point trying to keep them separate only adds confusion, duplication, etc. I did a lot of slash and burn in this PR.